db/view/view.cc

```diff
@@ -1787,7 +1787,7 @@ view_builder::view_builder(replica::database& db, db::system_keyspace& sys_ks, d
         , _sys_dist_ks(sys_dist_ks)
         , _mnotifier(mn)
         , _vug(vug)
-        , _permit(_db.get_reader_concurrency_semaphore().make_tracking_only_permit(nullptr, "view_builder", db::no_timeout, {})) {
+        , _permit(_db.get_reader_concurrency_semaphore(true).make_tracking_only_permit(nullptr, "view_builder", db::no_timeout, {})) {
```

Comment:
I think for the view builder, it doesn't really matter which semaphore we use, there's no unbounded concurrency there anyway, but I guess you're right, it can't hurt to use the "right" one.